### DISCUSSION TITLE
Занятие про Entity Framework core

### DISCUSSION THREAD

**Discussion by pingvin1308** [cleannetcode/CsharpForBeginers#6] [Матерые]
Привет, тут будут обсуждения по теме занятия. Даже если занятие прошло, все равно можно писать сюда вопросы и пожелания :)
Домашка:
1) Повторить подключение к БД и настройку БД через EF core. Настроить по примеру работу с DI
 Ветка для старта: https://github.com/pingvin1308/Timesheet/tree/homework-11-ef-core

2) Повторить реализацию репозитория EmpoloyeeRepository и подключение AutoMapper с настройкой маппингов в проекте. 
Ветка для старта: https://github.com/pingvin1308/Timesheet/tree/homework-12-ef-repositories%2Bautomapper

Также задача со звездочкой, для тех кто хочет двигать вперед, а не повторять)
Сделайте ef реализацию для ITimesheetRepository, чтобы текущая логика успешно работала

Выполненную домашку следует скидывать коментом в данной дискуссии)
И также рекомендую каждому из вас смотреть как другие выполнили задание, возможно даже давать совети и коментарии)
Вместе как говорится веселее, да и учиться будет проще ;)

К слову, правильно выполненную и проверенную домашку я помечаю какой нибудь реакцией 👍 и коментом, особенно если есть что поправить.

**Comment by artyomorel** [comment]
Отправляю выполненную домашку. Сделал:
1-Все по видео
2-Починил тесты (кроме IssuesServiceTest)
3-Добавил TimesheetRepository(Пришлось изменить немного модель Employee В базе данных)
4-Заменил все контроллеры по добавлению работников на один и создал для этого Request модель работника ( использовал так же Fluent Validation для проверки вводимой модели , постарался где можно использовать Automapper)
Ссылка на гитхаб: https://github.com/artyomorel/Timesheet/tree/homework-12-ef-repositories%2Bautomapper

**Reply by pingvin1308** [reply]
Шикарно все сделал, оставил пару комментов, но это почти незначительные мелочи)

**Reply by artyomorel** [reply]
Большое спасибо , комментарии доработаю
